test/local/pushbox.js

```diff
@@ -19,41 +19,40 @@ const mockConfig = {
     maxTTL: 123456000
   }
 }
-const mockDeviceIds = ['bogusid1', 'bogusid2', 'bogusid3']
+const mockDeviceIds = ['AAAA11', 'BBBB22', 'CCCC33']
 const mockData = 'eyJmb28iOiAiYmFyIn0'
-const mockUid = 'myuid'
-const pushboxModulePath = `${ROOT_DIR}/lib/pushbox`
+const mockUid = 'ABCDEF'
+
+const mockPushboxServer = nock(mockConfig.pushbox.url, {
+  reqheaders: {Authorization: `FxA-Server-Key ${mockConfig.pushbox.key}`}
+}).defaultReplyHeaders({
+  'Content-Type': 'application/json'
+})
 
 describe('pushbox', () => {
+
+  afterEach(() => {
+    assert.ok(nock.isDone(), 'there should be no pending request mocks at the end of a test')
+  })
+
   it(
     'retrieve',
     () => {
-      const FakePool = function() {}
-      const getSpy = sinon.spy(() => Promise.resolve({
-        status: 200,
-        last: true,
-        index: '15',
-        messages: [{
-          index: '15',
-          // This is { foo: "bar", bar: "bar" }, encoded.
-          data: 'eyJmb28iOiJiYXIiLCAiYmFyIjogImJhciJ9'
-        }]
-      }))
-      FakePool.prototype.get = getSpy
-      const mocks = {
-        './pool': FakePool
-      }
-      const pushbox = proxyquire(pushboxModulePath, mocks)(mockLog(), mockConfig)
-
+      mockPushboxServer.get(`/v1/store/${mockUid}/${mockDeviceIds[0]}`)
```

Comment:
These tests are pretty much all the same, but re-written to use `nock` like the customs-server ones, rather than `proxyquire`.  If we'd prefer to use `proxyquire` then I can do the reverse change and make the customs-server use that instead of `nock`, but I find the `nock` versions a bit easier to follow.

Reply:
@vladikoff may have an opinion on this, IIRC he was the person who originally suggested we use proxyquire.